src/Makevars

```diff
@@ -1,2 +1,2 @@
-GIT_COMMIT_INFO:=$(shell git log --pretty=oneline -1)
-PKG_CPPFLAGS=-I$(R_HOME)/src/include/ -DGIT_COMMIT_INFO="\"$(GIT_COMMIT_INFO)\"" --std=c++11
+GIT_COMMIT_INFO := $(shell git log --pretty=oneline -1)
```

Comment:
I want the commit message as well because that's more relatable. Having a commit hash suffices but we have to look it up for the corresponding commit message to get a sense of the version of the tracer.
Thanks, `!=` does the job. `make check` does not show any warnings now.